packages/experiments/src/components/M365Nav/M365Nav.tsx

```diff
@@ -0,0 +1,237 @@
+﻿/* tslint:disable */
+import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/components/FocusZone';
+import { Icon } from 'office-ui-fabric-react/lib/components/Icon';
+import { INavLinkGroup } from 'office-ui-fabric-react/lib/components/Nav';
+import { INavState } from 'office-ui-fabric-react/lib/components/Nav/Nav.base';
+import { AnimationClassNames } from 'office-ui-fabric-react/lib/Styling';
+import * as React from 'react';
+import {
+  IM365NavProps,
+  IM365NavLink,
+  IM365NavStyleProps,
+  IM365NavStyles
+} from './M365Nav.types';
+import {
+  getStyles
+} from './M365Nav.styles';
+import { M365NavBase } from './M365NavBase';
+import {
+  styled,
+  classNamesFunction
+} from 'office-ui-fabric-react/lib/Utilities';
+
+const getClassNames = classNamesFunction<IM365NavStyleProps, IM365NavStyles>();
+
+class M365NavComponent extends M365NavBase {
+  constructor(props: IM365NavProps) {
+    super(props);
+
+    this.state = {
+      isLinkExpandStateChanged: false,
+      selectedKey: props.initialSelectedKey || props.selectedKey
+    };
+  }
+
+  public render() {
+    if (!this.props.groups || this.props.groups.length === 0) {
+      return null;
+    }
+
+    return (
+      <FocusZone direction={ FocusZoneDirection.vertical }>
+        <nav role='navigation'>
+          {
+            this.props.groups.map((group: INavLinkGroup, groupIndex: number) => {
+              return this.renderGroup(group, groupIndex);
+            })
+          }
+        </nav>
+      </FocusZone>
+    );
+  }
+
+  private onLinkClicked(link: IM365NavLink, ev: React.MouseEvent<HTMLElement>): void {
+    var nextState: INavState = {
+      selectedKey: link.key
+    };
+
+    var hasChildren = link.links && link.links.length > 0;
```

Comment:
vars are not allowed; use `let` or `const`

Reply:
changed vars to let or const